app-admin/terraform/terraform-0.11.11.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit golang-vcs-snapshot
+
+DESCRIPTION="A tool for building, changing, and combining infrastructure safely"
+HOMEPAGE="https://www.terraform.io/"
+
+EGO_PN="github.com/hashicorp/${PN}"
+SRC_URI="https://github.com/hashicorp/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="pie"
+
+DOCS=( src/${EGO_PN}/CHANGELOG.md src/${EGO_PN}/README.md )
```

Comment:
Not a good idea to put variables here as well

Reply:
Probably drop docs then, users are going to have to look up documentation for plugins anyways